### ISSUE TITLE
can't get same result on FLD dataset

### ISSUE THREAD

**Issue by adkAurora** [shumming/GLE_FLD#5]
i use your code and annos to train your model on dataset FLD
but get very poor result , i didn't change anything that is important
and the loss isn't get small after first epoch

|  L.Collar  |  R.Collar  |  L.Sleeve  |  R.Sleeve  |   L.Waist  |   R.Waist  |    L.Hem   |   R.Hem    |     ALL    |
|   0.96064  |  1.06667  |   0.47118  |   0.95785  |   0.52165  |   0.80711  |   0.24167  |   0.65336  |  0.71002 |

**Comment by suminlee94** [comment]
You should train the network without non-local block at the begining of the training.
Please refer to 'Non-local Neural Networks' published in CVPR 2018.
